setup.py

```diff
@@ -49,7 +49,7 @@
                                           "apps/data/sample_data/*"]},
     "include_package_data": True,
     "description": "Open source library for continuous-variable quantum computation",
-    "long_description": open("README.rst", encoding="utf-8").read(),
+    "long_description": open("README.md", encoding="utf-8").read(),
     "long_description_content_type": "text/x-rst",
```

Comment:
I think this line should also be updated, otherwise PyPI will continue to try parsing it as a ReST file

Reply:
See https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/ (I think it should be `text/markdown` if I am reading it correctly?)

Reply:
Nice, thanks @josh146. Read through the link and updated it!